docs/get-started.md

```diff
@@ -67,25 +67,12 @@ react-native link react-native-square-in-app-payments
 To use the In-App Payments plugin on iOS devices, install **In-App Payments SDK for iOS** 
 to make it an available resource for the React Native plugin. 
 
-1. Open your iOS project `myRNInAppPaymentsSample.xcodeproj` with **Xcode** and install **In-App Payments SDK** by following  
-[Install In-App Payments SDK - Swift(iOS) Option 3: Manual Installation]. You **MUST**
-put the **SquareInAppPaymentsSDK.framework** in folder `<YOUR_PROJECT_DIRECTORY>/ios`, otherwise
-the project won't compile.
-    > Check the supported SDK version in the top of [root README].
+1. Open your iOS project `myRNInAppPaymentsSample.xcodeproj` with **Xcode** and install **In-App Payments SDK** by adding the following to your Podfile:
+  ```
```

Comment:
I believe we want to add here `use_native_modules!` as a requirement as well

Reply:
I had not seen the latest change, this changes my request, but use_frameworks! is not needed anymore when using Cocoapods 1.5.x and above and has conflicts with some React Native modules.

Reply:
https://github.com/square/in-app-payments-react-native-plugin/pull/25

Reply:
Great, that was the step I had issues with and that change fixes it perfectly

Reply:
Merged, thanks for confirming!